WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -82,7 +82,7 @@ export default function Delegates() {
   }
 
   return (
-    <div className="container">
+    <div style={{ margin: '16px' }}>
```

Comment:
Veto about including magic numbers in our CSS. We simply don't do that.
You can change the (old, pre-change) `<div className="container">` into an actual SemUI React `<Container>` and then play around with `<Container fluid>` or whatever else SemUI has to offer. But no hard-coded margins at any cost.

Reply:
Thanks a lot @gregorbg, <Container fluid> did the magic. :-)